src/frontend/dash/circlePacker.ts

```diff
@@ -1,12 +1,19 @@
 // note: circle radii are normalized on render
-const STEP_DISTANCE = .02
-const MARGIN = .14
+//const MARGIN = .14
+const MARGIN = 1.3;
+let aspectRatio = 1; // 2:1
+let deltaAvg = 0.95; // How much effect the avgeraging has on the circles
```

Comment:
I think that adding comments explaining exactly how these constants affect the final packing would be helpful

Reply:
Your new comments still don't explain how specifically they affect things. I know that deltaAvg changes how much effect the averaging has, but it doesn't explain what 0.95 means or what changing it to 7 would mean